docs/architecture/blazor-for-web-forms-developers/architecture-comparison.md

```diff
@@ -8,6 +8,26 @@ ms.date: 09/11/2019
 
 # Architecture comparison of ASP.NET Web Forms and Blazor
 
+While ASP.NET Web Forms and Blazor have many similar concepts, they also have difference in how they work. Here we will take a look at the underlying workings and architecture of ASP.NET Web Forms and Blazor to see how they compare.
+
 ## ASP.NET Web Forms
 
-## Blazor
\ No newline at end of file
+TODO
+
+## Blazor
+
+Blazor is client-side web UI framework similar in nature to JavaScript frontend frameworks like Angular or React. The Blazor framework deals with handling of user interactions and rendering UI updates accordingly. Blazor is *not* based on a request-reply model. User interactions are handled as events that are not in the context of any particular HTTP request.
```

Comment:
I thought that if hosted on the server the connection is within a request context that initiated the connection.

Reply:
Technically, yes, that is true, but the request to establish the connection is really an implementation detail of how Blazor Server apps deal with the user interactions. You don't interact with the request context in a Blazor Server app and the individual requests are not logically part of any particular HTTP request.